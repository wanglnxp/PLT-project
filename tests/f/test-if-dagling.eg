int main()	{
	if (3>2){
		print("true");
		if(3<4){
			print("3<4");
		}
	}
	if (3>6){
		print("second if");
	}
	else{
		print("false");
	}
	return 0;
}
