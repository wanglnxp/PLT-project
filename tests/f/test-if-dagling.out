true
3<4
false
